packages/utilities/psammead-assets/README.md

```diff
@@ -0,0 +1,29 @@
+# psammead-assets &middot; [![GitHub license](https://img.shields.io/badge/license-Apache%202.0-blue.svg)](https://github.com/BBC-News/psammead/blob/latest/LICENSE) [![npm version](https://img.shields.io/npm/v/@bbc/psammead-assets.svg)](https://www.npmjs.com/package/@bbc/psammead-assets) [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](https://reactjs.org/docs/how-to-contribute.html#your-first-pull-request)
+
+This package is a collection of variable constants which can be imported into your application.
+
+## Installation
+
+```jsx
+npm install @bbc/psammead-assets --save
+```
+
+## Usage
+
+```jsx
+import { $BBC_BLOCKS } from "@bbc/psammead-assets/svgs"
```

Comment:
May be a question for #45, but should the path here be so closely tied to technical implementation? Is it an `@bbc/psammead-assets/icon` instead of a `@bbc/psammead-assets/svgs`?